src/FluentlyHttpClient/Utils/QueryStringOptions.cs

```diff
@@ -98,6 +100,18 @@ public QueryStringOptions WithValueFormatter(Func<object, string> configure)
 		return this;
 	}
 
+	/// <summary>
+	/// Gets or sets the function to format a value/key. This will allow you to manipulate the value by key
+	/// </summary>
+	/// <param name="configure"></param>
+	/// <returns></returns>
+	public QueryStringOptions WithKeyValueFormatter(Dictionary<string, Func<object, string>> configure)
```

Comment:
This only formats the Value but per Key correct? e.g. it doesn't allow to format the key itself

if so 



```suggestion
	/// Gets or sets the function to format a value per key. This will allow you to manipulate the value by key
	/// </summary>
	/// <param name="configure"></param>
	/// <returns></returns>
	public QueryStringOptions WithValuePerKeyFormatter(Dictionary<string, Func<object, string>> configure)
```

Reply:
yep, renamed accordingly, thanks